src/ui/analytics.js

```diff
@@ -97,6 +103,30 @@ const analytics: Analytics = {
       });
     }
   },
+  emailProvidedEvent: () => {
+    if (analyticsEnabled && isProduction) {
```

Comment:
Making a `sendReactGAEvent` or equivalent function would eliminate these repeated if checks and could also help ensure the checks are not accidentally left out in the future.

Reply:
Yeah I've been meaning to do this, just did and it's way nicer now.